src/components/editor/EditorArticleController.jsx

```diff
@@ -119,6 +119,17 @@ export default class EditorArticleController extends FalcorController {
       // The update wasn't due to a change in article
       this.debouncedHandleFormStateChanges();
     }
+    // scroll to article editor once page loads
+    const { hash } = window.location;
```

Comment:
Nice, you have good knowledge of ES6 :)

Reply:
I agree with the comments. Didn't find the Router code at first glance, thanks for pointing me to it! Will update later tonight or tomorrow.

Reply:
Sounds good, just write a comment on the PR when you're ready for another review

Reply:
Or even better, if we're trying to enforce the Zenhub workflow I just did what I should've done and moved the issue and PR back to in progress, and when you're ready for review just move them back into Review/QA